gym/utils/seeding.py

```diff
@@ -23,6 +23,12 @@ def np_random(seed: Optional[int] = None) -> Tuple["RandomNumberGenerator", Any]
 # TODO: Remove this class and make it alias to `Generator` in a future Gym release
 # RandomNumberGenerator = np.random.Generator
 class RandomNumberGenerator(np.random.Generator):
+    def __eq__(self, other):
```

Comment:
Q: Does `np.random.Generator` implement `__eq__`? If so, then shouldn't it also be called here?

Reply:
Why is this marked as resolved?